website/content/docs/nia/license.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Consul-Terraform-Sync Enterprise License
+description: >-
+  Consul-Terraform-Sync Enterprise License
+---
+
+# Consul-Terraform-Sync Enterprise License
+<EnterpriseAlert>
+  Licenses are only required for Consul-Terraform-Sync Enterprise
+</EnterpriseAlert>
+
+Consul-Terraform-Sync Enterprise binaries require a license in order to start. The required license must be a [Consul Enterprise license](/docs/enterprise/license/overview) as there is no specific Consul-Terraform-Sync Enterprise license. As a result, Consul-Terraform-Sync Enterprise's licensing is very similar to Consul Enterprise. Consul-Terraform-Sync Enterprise is available with all its features with any valid, unexpired Consul Enterprise license regardless of its packaging/pricing model or level.
```

Comment:
The content is good though I found it verbose because the CTS ent name is long and very similar to Consul Enterprise. I took a jab at shortening this paragraph by reducing the CTS Ent occurrence from 4 to 3.

```suggestion
Consul-Terraform-Sync Enterprise binaries require a [Consul Enterprise license](/docs/enterprise/license/overview) to run. There is not a specific license type for Consul-Terraform-Sync Enterprise. As a result, licensing is very similar to Consul Enterprise. 

All features offered in Consul-Terraform-Sync Enterprise is available with any valid, unexpired Consul Enterprise license. It is not dependent on your Consul Enterprise  packaging or pricing model.
```

Reply:
Thank you for shortening this. I think it's more concise and to the point. I do think the last sentence about availability by packaging/pricing is helpful since certain Consul Enterprise features are available with certain tiers and wanted to be explicit that CTS is different and available regardless of tier. Let me try rewriting the sentence to make it more concise.

:nod: that CTS Ent name is so long

Reply:
I'm realizing just now that you moved the sentence about the Consul packaging/pricing to its own paragraph 🤦‍♀️ . Sorry for not reading carefully before replying 🙇 . The diff for this change can be viewed: https://github.com/hashicorp/consul/pull/10803/commits/2d8c2bf12f81aa86344761ba3d83b32e989659d0

Reply:
No worries! I thought it was an important sentence too, deserving its own paragraph :D